core/init.php

```diff
@@ -11,51 +11,55 @@
 
 // Nameless error handling
 require_once(join(DIRECTORY_SEPARATOR, array(ROOT_PATH, 'core', 'classes', 'ErrorHandler.php')));
-set_error_handler("ErrorHandler::catchError");
-register_shutdown_function("ErrorHandler::catchFatalError");
+set_exception_handler([ErrorHandler::class, 'catchException']);
```

Comment:
Does this replace the need for `register_shutdown_function`?

Reply:
Good point, I misunderstood the documentation. 
I have added it back with a new function.